lib/Net/SSLeay.pod

```diff
@@ -3810,8 +3810,23 @@ Returns a function pointer to the TLS/SSL method set in $ssl.
 
 Check openssl doc L<http://www.openssl.org/docs/ssl/SSL_CTX_set_ssl_version.html|http://www.openssl.org/docs/ssl/SSL_CTX_set_ssl_version.html>
 
+=item * in_init, in_before, is_init_finished, in_connect_init, in_accept_init
+
+B<COMPATIBILITY:> not available in Net-SSLeay-1.85 and before.
```

Comment:
"Requires Net-SSLeay x.xx or above" would be easier to parse, but I guess we don't know the next stable version number yet. Maybe we should decide on a stable/unstable version numbering scheme, like ExtUtils-MakeMaker has.

Reply:
This follows the convention that the existing compatibility notes have. I took a look at the existing compatibility notes and they already state the minimal ssl library version requirement. Changing these to minimal Net-SSLeay would be a simple search-and-replace operation.